M6-Backend/FacilityInterfaces/interfaceAddFacility.js

```diff
@@ -0,0 +1,42 @@
+
+function interfaceAddFacility(Info){
+    var newId = Info.id;
+    var title = Info.title;
+    var description = Info.description;
+    var long = parseFloat(Info.long);
+    var lat = parseFloat(Info.lat);
+    var type = Info.type;  
+    var facilityImageLink = Info.facilityImageLink;
+
+    var finalResult;
+
+    if(newId == null || title == null || description == null || long == null || lat==null
+        || type == null || facilityImageLink == null){
+            return "Error:Invalid add, missing field";
+        }else{
+            finalResult ={
+                id: newId,
+                facility:
+                {
+                    "facilityType": type,
+                    "facility_status": "normal",
+                    "facilityTitle": title,
```

Comment:
Codacy has a fix for the issue: [Replace `····················"facilityTitle"` with `········facilityTitle`](https://app.codacy.com/gh/CPEN321HELP/CPEN321/pullRequest?prid=9852520)
```suggestion
        facilityTitle: title,
```